### DISCUSSION TITLE
划分段落

### DISCUSSION THREAD

**Discussion by Joker1212** [RapidAI/RapidOCRPDF#17] [Ideas]
1.pdf分段按各级标题组织很合理，难点会集中在标题的识别,找到了标题和层级，分段就很自然了
2.因为布局模型都是单页扫描，会遗漏跨页或者双列跨列的文段和表格，建议进行跨列跨页的识别框拼接，再进行识别
这是我使用rappidlayout进行分段的尝试，先进行了跨列合并，再进行了跨页合并
原始识别:<img width="659" alt="image" src="https://github.com/user-attachments/assets/a1940e02-257b-4589-b64a-c5c0119c2760">
跨列合并:
<img width="611" alt="image" src="https://github.com/user-attachments/assets/7dc6e287-a543-4014-a118-f72d3fa92f45">
跨页合并:
<img width="603" alt="image" src="https://github.com/user-attachments/assets/604a4519-db15-4031-bea6-b9bbc66b6679">

**Comment by SWHL** [comment]
看着很不错，不懂就问：跨列合并怎么做到的呀
